.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  mix_test:
+    name: mix test (OTP ${{matrix.otp}} | Elixir ${{matrix.elixir}})
+    strategy:
+      matrix:
+        elixir: ['1.4.5', '1.5.1']
```

Comment:
I believe this value is not being used? I will try removing it!

Reply:
Nope, we need it! :D

Reply:
Correct. At moment with GitHub actions, the `includes` section is not additive like Travis. It acts as a modifier for configurations possible from the matrix.